README.md

```diff
@@ -1,24 +1,226 @@
-# README
+<a name="readme-top"></a>
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+  <img src="Kreative-Keeper.png" alt="logo" width="auto"  height="auto" />
+  <br/>
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+  <h3><b>Microverse README Template</b></h3>
```

Comment:
# [OPTIONAL]

- [ ] Kindly remove this text to  make your readme file clean

Reply:
Thank you. I have removed it.